commands/host.vegman

```diff
@@ -224,6 +224,86 @@ function cmd_virtualmedia_umount {
   usb-ctrl eject "usb.ms.cli"
 }
 
+
+function loc_config_default_set {
+  local yes=""
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      -y | --yes) yes="y";;
+      *) abort_badarg "$1";;
+    esac
+    shift
+  done
+
+  echo "Host BIOS settings will be reset to default on next boot."
+  if [[ -z "${yes}" ]]; then
+    confirm
+  fi
+
+  ipmitool chassis bootdev none clear-cmos=yes >/dev/null
+}
+
+function loc_config_default_cancel {
+  local yes=""
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      -y | --yes) yes="y";;
+      *) abort_badarg "$1";;
+    esac
+    shift
+  done
+
+  echo "Cancel reset to default host settings on next boot."
+  if [[ -z "${yes}" ]]; then
+    confirm
+  fi
+
+  ipmitool chassis bootdev none >/dev/null
+}
+
+function loc_config_default_status {
+  expect_noarg "$@"
+  strArr=($(ipmitool chassis bootparam get 5|grep -i 'Boot parameter data'))
+  val=${strArr[3]}
+  len=${#val}
+
+  if [[ $((16#${val:$len-8:1})) -ge 8 ]]; then
```

Comment:
Why all this magic? Why not just grep `ipmitool` output for `CMOS Clear` ? As far as I can see, that's what `ipmitool` would pring if the flag is set:

```c
if(rsp->data[3] & BF2_CMOS_CLEAR)                                 
        printf("   - CMOS Clear\n"); 
```

Reply:
You're still doing magic. Please either explain why, or switch to checking for "CMOS Clear" in the output of `ipmitool`.